.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ on:
 jobs:
   windows:
     name: Windows
-    runs-on: windows-2019
+    runs-on: windows-latest
```

Comment:
I would recommend that we are explicit about the version, and use `windows-2022`.
Also do we want to switch to the 2022 toolchain in our Azure Pipeline builds as well?

Reply:
> I would recommend that we are explicit about the version, and use windows-2022.

👍 

> Also do we want to switch to the 2022 toolchain in our Azure Pipeline builds as well?

It would be good to do as part of this PR, how do we make that change ?

Reply:
That needs a bit more work as we need to create a new agent pool using the 2022 image. 
Let's fix GH CI with this PR and then we can work on updating Azure Pipelines this week.